### ISSUE TITLE
motu-client not available in PATH after installation

### ISSUE THREAD

**Issue by fox91** [clstoulouse/motu-client-python#10]
Why should I use this command:
```bash
python <PATH_TO_MOTUCLIENT_DIR>/motu-client.py [OPTIONS]
# example:
python /tmp/ENV_MOTU/lib/python3.5/site-packages/motu-client.py [OPTIONS]
```
when I could simply use this?
```bash
motu-client [OPTIONS]
```

**Comment by clstoulouse** [comment]
Hi,

In fact it depends on the manner you have installed.

- Best way: https://github.com/clstoulouse/motu-client-python#UsagePIP

python -m motu-client -h  

- Alternative https://github.com/clstoulouse/motu-client-python#usage-from-targz-installation
You have to be in the folder where Motu has been extracted.

Regards,
Sylvain

**Comment by fox91** [comment]
@clstoulouse my solution could be acceptable?

**Comment by bakhansen** [comment]
@fox91 you could also add it to your path:

`export PATH="$PATH:/path/to/motu-client-python"`

after which you can execute `motu-client.py [options]`

**Comment by clstoulouse** [comment]
@fox91  Yes it is, we will merge your PR soon...

@bakhansen  Great tips !

**Comment by clstoulouse** [comment]
Dear all,

I have merged the PR11, so this issue is fixed in Motu v1.8.1.
Thanks a lot for your feedbacks.

Enjoy Motu !
